es/criteria/reusable-and-portable-codebases.md

```diff
@@ -29,25 +29,25 @@ order: 3
 * Construir una comunidad.
 * Trasladar la responsabilidad de la documentación, el soporte, la corrección de errores, etc. a otra parte.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"